Pixi/README.md

```diff
@@ -0,0 +1,15 @@
+Fable bindings for Pixi.js 
+=======
+
+### Includes
+- Fable bindings for [Pixi](https://github.com/pixijs/pixi.js).
+- Fable bindings for [pixi-sound](https://github.com/pixijs/pixi-sound).
+- Fable bindings for [pixi-particles](https://github.com/pixijs/pixi-particles).
+
+### Using
+1. `paket add Fable.Import.Pixi`
+2. `yarn add pixi.js pixi-particles pixi-sound`
```

Comment:
@whitetigle 

Didn't you say that all the libs was coming from the same module ?
Here you are adding one module per import. So, here you could use pixi without pixi-sound and pixi-particle.

So, it's should be 3 separate nuget packages.

Reply:
No that's not what I meant. But I may have misunderstood your point though 😉 

So,
Pixi does neither need pixi-sound nor pixi-particles, 
But Pixi-sound needs pixi and pixi-particles too.

So what am I to do?

Reply:
Ah ok :)

No problem, I didn't understood the right thing them :)

So you need to create 3 packages:
- Fable.Import.Pixi
- Fable.Import.PixiSound (or Fable.Import.Pixi.Sound)
- Fable.Import.PixiParticle (or Fable.Import.Pixi.Particle)

Then you can use, `ProjectReference` in the fsproj of PixiSound and PixiParticle to make then dependant of Pixi.
As an example: https://github.com/fable-compiler/fable-react/blob/master/src/Fable.ReactLeaflet/Fable.ReactLeaflet.fsproj#L10-L12

Then, in the `build.fsx` make sure to list the project in the correct order:

1. First Pixi
2. Then PixiSound and PixiParticle

This will ensure, that if we update Pixi and PixiSound for example then we release Pixi first and update it's version in the fsproj and so PixiSound will reference the right version of Pixi too.

Reply:
ok I'll do that tomorrow. Thanks @MangelMaxime 👍